README.md

```diff
@@ -39,12 +39,13 @@ $$
 You can solve an JuMP model `model` by using [NLPModelsJuMP.jl](https://github.com/JuliaSmoothOptimizers/NLPModelsJuMP.jl) to convert it.
 ```julia
 using JuMP, NLPModelsJuMP, Percival
-model = Model()
+model = Model(NLPModelsJuMP.Optimizer)
```

Comment:
This is possible now thanks to https://github.com/JuliaSmoothOptimizers/NLPModelsJuMP.jl/pull/157